simtools/util/validate_data.py

```diff
@@ -96,6 +109,50 @@ def _check_required_columns(self):
                     self._logger.error("Missing required column '{}'".format(key))
                     raise
 
+    def _sort_data(self):
+        """
+        Sort data according to one data column
+        (if required by any column attribute)
+
+        Data is either sorted or reverse sorted
+        """
+
+        _columns_to_sort = []
+        _columns_to_reverse_sort = []
+        for key, value in self._reference_data_columns.items():
+            if 'attribute' in value:
+                if 'sort' in value['attribute']:
```

Comment:
Not sure, this is a list search:
```
value=['reversesort']
if 'sort' in value:
	print('found sort')
else:
	print('sort not found')
```
results in 'sort not found'. So that seems to be fine.

I've tested this by adding a unit test to reverse sorting (commit 0590a3a)